sections/todo/time_tracker.js

```diff
@@ -313,35 +316,65 @@ var TimeTracker  = class TimeTracker {
         return false;
     }
 
-    _enable_file_monitors () {
-        [this.daily_csv_file_monitor,] = MISC_UTILS.file_monitor(this.daily_csv_file, () => {
-            this._on_tracker_files_modified();
-        });
-
-        [this.yearly_csv_file_monitor,] = MISC_UTILS.file_monitor(this.yearly_csv_file, () => {
-            this._on_tracker_files_modified();
-        });
-
-        [this.yearly_csv_dir_monitor,] = MISC_UTILS.file_monitor(this.yearly_csv_dir, () => {
-            this._on_tracker_files_modified();
-        });
+    _enable_file_monitors (timeout = 100) {
+        try{
```

Comment:
That try-catch seems redundant to me.

Reply:
It is not necessary. I used try-catch just because `MISC_UTILS.file_monitor` has a try-catch.